js/app/modules/filter/multi.js

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 Endless Mobile, Inc.
+
+/* exported Multi */
+
+const GObject = imports.gi.GObject;
+
+const Filter = imports.app.interfaces.filter;
+const HistoryStore = imports.app.historyStore;
+const Module = imports.app.interfaces.module;
+
+const Multi = new Module.Class({
```

Comment:
drive-by comment, I would much rather see this implemented the same way as we do with Order: https://github.com/endlessm/eos-knowledge-lib/blob/master/js/app/interfaces/order.js#L46